CoffeeChat/Cells/CommunityUserTableViewCell.swift

```diff
@@ -0,0 +1,147 @@
+        interestsCollectionView.register(
+            InterestTagCollectionViewCell.self,
+            forCellWithReuseIdentifier: InterestTagCollectionViewCell.reuseIdentifier
+        )
+        interestsCollectionView.layoutIfNeeded()
+        containerView.addSubview(interestsCollectionView)
+
+        setupConstraints()
+    }
+
+    private func setupConstraints() {
+        let bottomPadding: CGFloat = 8
+        let topPadding: CGFloat = 12
+
+        containerView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.bottom.equalTo(contentView).inset(bottomPadding)
+            make.top.equalToSuperview()
+        }
+
+        profileImageView.snp.makeConstraints { make in
+            make.leading.top.equalTo(containerView).offset(topPadding)
+            make.size.equalTo(CGSize(width: 36, height: 36))
+        }
+
+        nameLabel.snp.makeConstraints { make in
+            make.top.equalTo(profileImageView)
+            make.leading.equalTo(profileImageView.snp.trailing).offset(topPadding)
+            make.trailing.equalTo(containerView).inset(topPadding)
+            make.height.equalTo(20)
+        }
+
+        informationLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalTo(nameLabel)
+            make.height.equalTo(13)
+            make.top.equalTo(nameLabel.snp.bottom).offset(4)
+        }
+
+        interestsCollectionView.snp.makeConstraints { make in
+            make.leading.trailing.equalTo(nameLabel)
+            make.top.equalTo(informationLabel.snp.bottom).offset(bottomPadding)
+            make.bottom.equalTo(containerView).inset(topPadding)
+        }
+    }
+
+    func configure(with user: User) {
+        nameLabel.text = "\(user.firstName) \(user.lastName)"
+        informationLabel.text = "\(user.major) · \(user.graduationYear) · \(user.hometown) · \(user.pronouns)"
+        interests = user.interests
+        interestsCollectionView.reloadData()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+}
+
+extension CommunityUserTableViewCell: UICollectionViewDataSource, UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return interests.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: InterestTagCollectionViewCell.reuseIdentifier, for: indexPath) as?
+                InterestTagCollectionViewCell else { return UICollectionViewCell() }
+        let interest = interests[indexPath.row]
+        cell.configure(with: interest)
+        return cell
+    }
+}
+
+extension CommunityUserTableViewCell: UICollectionViewDelegateFlowLayout {
+    func calculateNecessaryWidth(text: String) -> CGFloat {
```

Comment:
Spacing at the beginning/end of extensions/classes is somewhat inconsistent throughout imo.